.gitignore

```diff
@@ -57,6 +57,9 @@ Carthage/Build
 
 fastlane/report.xml
 fastlane/screenshots
+.bundle/
+Gemfile.lock
```

Comment:
@sebalopez any reason why we should exclude the .lock file? This should ensures the env uses the same versions everywhere right?

Reply:
hmm I think this is here by mistake, will remove